gattc_nina.go

```diff
@@ -0,0 +1,111 @@
+//go:build nina || nano_rp2040
+
+package bluetooth
+
+type uuidWrapper = UUID
```

Comment:
Is there a reason why you define this type alias?

Reply:
Changed to just use `UUID` type.